### DISCUSSION TITLE
How can I switch from `conflicting-main-...` back to `main`

### DISCUSSION THREAD

**Discussion by MaxG87** [zeapo/Android-Password-Store#2843] [Q&A] [answered]
Some days ago I tried forgot to `git pull` before I created a new password using the Android Passwordstore App. As a result I seem to have gotten a _conflicting rebase_ situation. I am certain that the remote `main` is in the desired state and would like to simply tell the app to just continue to use it.

I tried to click the _hard reset to remote branch_ button a few times, but to no avail. I then gave in and clicked _abort rebase and push new branch_. After that, a new branch appeared, `conflicting-main-1704023617699`. Since I didn't want to use its contents, I removed it using Git on a Linux desktop: `git push --delete origin conflicting-main-1704023617699`.

The app told me that after resolving the situation, it would continue to use `master`. I read `master` to be meant "whatever is the default branch".  I hoped that the app would recognise things as resolved if the branch `conflicting-main-1704023617699` disappeared. Apparently, that hasn't happened.

Now I am in a situation that I cannot receive new passwords. Also, I am unable to switch branches. I would like to know what I could do to resolve this situation.

**Comment by msfjarvis** [comment, marked as answer]
v1.13.5 has issues with changing branches since it tries to infer your default branch automatically and occasionally fails to do so, this has been changed for the next major release to prompt the user for a branch name instead.

If you know your remote branch is in the desired state and there is no data that only exists on the device then deleting the local repository and cloning it again is the simplest option.

**Reply by MaxG87** [reply]
Thank you very much. It was very straight-forward and easy to do.

Initially, I refrained from that option because I feared I had to reenter all the repository's details again, like URI, username, SSH-Key etc. **However, the App remembered all of that!** Giving that, everything was super convenient and no "other" option is really required.
